TwitchLib.Client/TwitchClient.cs

```diff
@@ -1171,13 +903,12 @@ private async Task HandlePrivMsg(IrcMessage ircMessage)
                 await OnUserIntro.Invoke(this, new() { ChatMessage = chatMessage });
             }
 
-            if (ChatCommandIdentifiers.Count != 0
-                && !string.IsNullOrEmpty(chatMessage.Message)
-                && ChatCommandIdentifiers.Contains(chatMessage.Message[0])
-                && OnChatCommandReceived != null)
+            if (ChatCommandIdentifiers.Contains(chatMessage.Message[0]))
             {
-                await OnChatCommandReceived.Invoke(this, new() { Command = new ChatCommand(chatMessage) });
+                _ = CommandInfo.TryParse(chatMessage.Message.AsSpan(), out commandInfo);
```

Comment:
I kinda liked `OnChatCommandReceived` event. It was easier to handle on user-side. Now I have to do if checks to decide if message is/isn't command.
IMHO I wouldn't remove that event and just do if-statement here. And either invoke MessageReceived or CommandRecieved.

Reply:
>It was easier to handle on user-side.

it depends on your point of view
Cons -
- you have to do an `if` check to see if it's a command

Pros +
- the code is easier to maintain
- you can simply share logic for processing commands
- in some cases, you don't have to subscribe to 4 events, but only to 2 events

Reply:
I'm in the camp of @Bukk94 here, it's nice to have a separate handler specifically for chat commands that won't get invoked when receiving any `PRIVMSG`, I think it lends itself to a more performant and simpler design from the standpoint of user code if we have a separate `OnChatCommandReceived` event.